### PR TITLE
Recreate awesome books with ES6

### PR DESCRIPTION
**Pull request by sanieni6**
# Recreate awesome books with ES6 🤓 
In this project I added the next features:
+ Re-create my Awesome books using ES6 syntax  🚀 
+ Divide my code into modules. 🗃️ 
+ Display the current date below the navigation bar using luxon library ⏳

### CODE REVIEW

File: modules/currentTime.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-return-assign */
+import { DateTime } from './luxon.js';
+
+export const displayTime = document.querySelector('.current-time');
+
+export function time() {
+  const dt = DateTime.local();
+  return displayTime.textContent = dt.toFormat('MMMM d, yyyy - hh:mm:ss a');
+}
```
Comment (thecodechaser) [inline modules/currentTime.js]: - [ ] Please change this function to an arrow function to follow the ES6 syntax as requested in the project requirements.
  - You can consider using the below code.
```ruby
export const time = () => {
  const dt = DateTime.local();
  return displayTime.textContent = dt.toFormat('MMMM d, yyyy - hh:mm:ss a');
}

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @sanieni6👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Highly organized code. ✔️
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sanieni6** [comment]
Recreate awesome books with ES6 🤓
Changes applied:
+ change the function to an arrow function to follow the ES6 syntax as requested in the project requirements.

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @sanieni6, 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Your PR has a good title and a well written summaty. ✔
- Your commit messages are descriptive. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**